src/config.rs

```diff
@@ -99,6 +101,8 @@ pub struct Args {
     pub elasticsearch: Option<ElasticsearchConfig>,
     /// See `sinks::Firehose` for more.
     pub firehosen: Option<Vec<FirehoseConfig>>,
+    /// See `sinks::Kinesis` for more.
+    pub kinesises: Option<Vec<KinesisConfig>>,
```

Comment:
Heh, 'firehosen' and 'kinesises'.

Reply:
We're very good at plurals, @pulltab.

Reply:
That changeset made me feel like Smeagol.

Reply:
:)